client/components/UserNicknamesPanel.js

```diff
@@ -4,55 +4,87 @@ import React from 'react'
 
 
 
-
 // Component imports
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 import { connect } from '../store'
 import { selectUser } from '../store/selectors'
 import AddNicknameForm from './AddNicknameForm'
 // import ConfirmActionButton from './ConfirmActionButton'
 import Component from './Component'
 
-
+// Component constants
+const MAXNICKS = 20 // Maximum IRC Nicknames allowed
 
 
 
 @connect
 class UserNicknamesPanel extends Component {
-  _deleteNickname = async (event) => {
+  _handleDeleteNickname = async (event) => {
     await this.props.deleteNickname(event.target.name)
     return 'Deleted!'
   }
 
+  state = {
+    formOpen: false,
+  }
+
+  _handleToggle = () => {
```

Comment:
I'd rename this to something more descriptive like
```suggestion
  _handleFormVisibilityToggle = () => {
```